src/components/Consents/Consents.test.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import Consent from 'components/Consents';
+import 'i18NextInit';
+
+const store = (
+  loading = false,
+  error = '',
+  definitions = [],
+  labels = [],
+  checked = []
+) => ({
+  consents: {
+    definitions,
+    labels,
+    checked,
+    loading,
+    error
+  },
+  publisherConfig: { publisherId: '' }
+});
+
+const middleware = [thunk];
+const mockStore = configureStore(middleware);
+
+describe('PaymentInfo component', () => {
+  test('should render GeneralErrorStyled (if loading = false)', async () => {
+    const { getByText } = render(
+      <Provider store={mockStore(store(false, 'noPublisherId'))}>
+        <Consent />
+      </Provider>
+    );
+
+    expect(
+      getByText(
+        /Unable to fetch terms & conditions. Publisher is not recognized/i
+      )
```

Comment:
I'm not sure if checking the text is ok here. If we will change the copy in the future it will break the test. We could also check if GeneralErrorStyled is rendered